util/throttle.cc

```diff
@@ -30,6 +30,7 @@
 #include "util/cache2.h"
 #include "util/db_list.h"
 #include "util/flexcache.h"
+#include "util/hot_threads.h"
 #include "util/throttle.h"
 
 namespace leveldb {
```

Comment:
For lines 38-39, I recommend changing from using raw pthread_mutex_t and pthread_cond_t raw types to the port::Mutex and port::CondVar types; then use scoped leveldb::MutexLock objects in place of explicit pthread_mutex_lock()/pthread_mutex_unlock() calls.